crates/nuon/README.md

```diff
@@ -0,0 +1,50 @@
+Support for the NUON format.
+
+The NUON format is a superset of JSON designed to fit the feel of Nushell.
+Some of its extra features are
+- trailing commas are allowed
+- quotes are not required around keys
+- comments are allowed, though not preserved when using [`from_nuon`]
+
+## Example
+below is some data in the JSON format
+```json
+{
+    "name": "Some One",
+    "birth": "1970-01-01",
+    "stats": [
+      3973513661747655316,
+      5120696105336210269,
+      7051687507382583899,
```

Comment:
I think this is helpful and a great example, but I don't think we need 11 entries in the stats list to help people understand. I think 3 would suffice.

Reply:
aha yeah i got carried away it appears :rofl:

Reply:
fixed in afd009fac